.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: ruby
 rvm:
   - 2.2.3
+before_script:
+  - sed -ie '/baseurl/s/\/m-lab\.github\.io//' _config.yml # sets baseurl to "" in _config for deploying w/o unnecessary url paths
```

Comment:
Can we put the comment on the previous line? I'd like to avoid exceeding 80 columns when possible.